SU2_CFD/src/output/CNEMOCompOutput.cpp

```diff
@@ -254,6 +254,8 @@ void CNEMOCompOutput::SetVolumeOutputFields(CConfig *config){
     if (nDim == 3)
      AddVolumeOutput("SKIN_FRICTION-Z", "Skin_Friction_Coefficient_z", "PRIMITIVE", "z-component of the skin friction vector");
 
+    AddVolumeOutput("THERMAL_COND_TR", "Thermal_Cond_tr", "PRIMITIVE", "thermal tr");
+    AddVolumeOutput("THERMAL__COND_VE", "Thermal_Cond_ve", "PRIMITIVE", "thermal vib");
```

Comment:
Use more descriptive descriptions please
```suggestion
    AddVolumeOutput("THERMAL_COND_VE", "Thermal_Cond_ve", "PRIMITIVE", "thermal vib");
```

Reply:
This was not resolved, the descriptions don't say anything meaningful

Reply:
Ahhh my bad.  My pre-coffee brain was not working.  It should be updated now.